go.mod

```diff
@@ -1,14 +1,20 @@
-module github.com/markbates/goth
+module github.com/damonkeys/goth
+
+go 1.15
 
 require (
+	cloud.google.com/go v0.63.0 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/gorilla/mux v1.6.2
 	github.com/gorilla/pat v0.0.0-20180118222023-199c85a7f6d1
 	github.com/gorilla/sessions v1.1.1
 	github.com/jarcoal/httpmock v0.0.0-20180424175123-9c70cfe4a1da
 	github.com/lestrrat-go/jwx v0.9.0
 	github.com/markbates/going v1.0.0
+	github.com/markbates/goth v1.65.0
```

Comment:
This is probably an artifact due to line above

Reply:
should be resolved due to 59a4506cdec732fef480e4bac5a34a3550836cdb